src/components/RestaurantBox.js

```diff
@@ -0,0 +1,31 @@
+import "./RestaurantBox.css";
+import Picture from "../img/restaurant1.png";
```

Comment:
Picture is not a component, so to avoid confusion let's call it just something like `restaurantImgSrc` or similar.
let's  just use the capitalized names for Components.